examples/pypi/helium_test.py

```diff
@@ -0,0 +1,5 @@
+from helium import start_chrome, write, press, ENTER
+start_chrome('prefix.dev')
```

Comment:
Hmm, that looks like one of the heaviest sdists to test. Does it download chrome? Shouldn't we try something slimmer?

Reply:
If you have any ideas please put them in here!

Reply:
https://pypi.org/project/bs4
https://pypi.org/project/future

Reply:
There is also https://pypi.org/project/sdist/#files

ChatGPT wrote me a script to find all sdists only pacakges:

```
pyupdog
PyAppleSMS
hrpy
ros-cdk-ecs
openlabs_nereid_webshop
dean_wade_nester
openweathermap_requests
quantlw-sdk
soso-python
meta-example
dl-helper
pdegen
interface-meta
jsontofu
ase
omarbasiccalculator
influxer2
django-inet
netease-musicbox
urwidtrees
au-nz-jobs
emencia-django-slideshows
softfire-sdk
gaussian-compute
ax-tools
rokcetshipy
robotframework-xvfb
xenum
hardscan
django-fusionbox
psycopg2cffi-compat
carotte
openrepos-webclient
nbeats-forecast
salientregions
tox-setuptools
await-cmd
mal-scraper
list-printer-package
CycLog
yoconfigurator
FuzzManager
plussize
gera2ld-pyserve
double12
trait_documenter
jaraco.packaging
ASCE
simpledistributions
txairbrake
aioconcurrency
djangorestframework-authentication
alfredo-py
gpswe
myffmpeg
cloudmesh_pbs
fcis
scanoramaCT
shrun
fake-agent
Flask-Pika
Sentosa-SQL-Library
scihub-cn
Liyi
mypy-boto3-lex-runtime
rapydcss
CSecurityBear
mygraph
aarpy
pygeoguz
iload
chef-cli
something
lovelace
term-chatgpt
vproces
apple-distributions
asterisklint
neon-solvers
vortext
mcrpc
Kstructure
tfmpl
sixtrackwrap
gazest-extra-macros
mudrex
oneto
pytgram
frigcal
imagify-python
duckdb
EntityStore
bgajjela
tirf-ob
drequests
yabgp
drupy
linex
python-cryptapi
sorbet
pycopy-zipimport
reblok
streamlit-jina
z3c.builder.core
murpy
MyPipModule
modbus-wrapper
pequests
swacch
setupbox
conjugate-prior
suggestscraper
rummy
Diana-Mice
kata
saltant-py
PufferFish
aminebasiccalculator
Flask-Diamond
twitch_chat_wrapper
DRF-commands
ensemble-networkx
tornadotoad
django-iprir
obj2html
xhj-nameko-dependency
siterank
knnimpute
CIRpy
pdf-renderer
ccc-gistemp
easy-web-app
deep-nccl-wrapper
pdtweak
engineering-calculator
lyricCheck
altin
jismesh
qtile
waste_not_the_water
BossLike
so-lazy
cipher
ProjectB-clientside-template-package
hmt-basemodels
estimotty
geosupport-suggest
google-reverse-search
TabBrasil
drf-model-pusher
randomization
shortcutml
crucyble
flet-restyle
diffpy.mpdf
Finance-Ultron
sentry_elastic_nodestore
pydataset
mit
masahiro-util
mini-redis
netcat-ntt
pdftotext
Foodle
no-error-python-shell
pygcam
QSymbolic
dn-bpl
cldclr
oydiv_rpc
TkEngine
mqreceive
KezMenu
astride
apilib
py-tools-ds
python-TAMER
twitter-auto-follow-bot
unoserver
django-usuario
django-oneall
tabel
nertivia
plotclassification
tulflow
pysteps
DiscordBotsTK
bigfeta
funcoperators
dupfilefind
GeobricksGeocoding
Products.whoopass
UW-RestClients-NWS
ndg
scratch-dir
nesterliuqin
pynv
pywizard-extra-modules
rma
Products.jsImagePopups
django-vkontakte-wall
HackerNews
flexer
jefrysaludo
gapic-google-cloud-pubsub-v1
django-featured-item
allen
vis-sample
Django-Pushy
RevImg
BinPackerPro
DamPy
surveymonster
nistbeacon
pyfanotify
mediamosa
snipeitv2
csvsorter
dash-google-charts
delugeonal
atb
textutils
gclient-service-account-auth
pyblish-houdini
pylandroid
sphinx-pyreverse
visaplan.zope.reldb
koodous
holtz-tools
js.jquery_slimbox
pypacking
Mopidy-IntergalacticFM
django-mp-barcode
dynamic-watershed
brewblox-stepper
pymagnolia
pyppl
Products.Clouseau
PythonLoopback
jhhalchemy
yasiu-vis
gimble
tempocalculation
django-kongconsumer
js.chosen
pynetfilter_conntrack
libjournal
twitter-ads-novalytica
fmp-x
sos4py
madxpy
contrive-orm
ebooks
smartaddress
diameter-synthesis
markusapi
imgvision
drf-rehive-extras
django-totalsum-admin
LineToolkit
django-changelog
Flask-Landing
repoze.zope2
bakerman
pymaestro
3d-wallet-generator
pyramid_simpleform-boldprogressives_fork
Products.MailTemplates
kwalitee
trade-bundle
reqlibpy
modularjs
amenu
overlay-parse
TOPSIS-PARNEET-101803075
errormiddleware
```

Reply:
```
s2-data
django-ajax-filtered-fields
wpath
BDT
abna
xzfpy
fireblocks-sdk
metacards_bert
zc.time
pyobjc-framework-XgridFoundation
die
CorePost
optzer
nesterfaund
woe-monotonic-binning
luca
geo-measures
inquant.contentmirror
opycleid
can
webview
lyloou-nester
xmlsec
linearregressor
mead-baseline
django-picker
sqlwrapper
qtpad
rq-scheduler
histodata
pd-parser
potodo
deepthought
xonsh-scrapy-tabcomplete
mate-shell
zynlp
syntactic-tagger
video-frame-selector
susanscleaner
awesome-dblp
iml
ftptool
jeb
better_verilog
pkgen
fuzzyfinder
StringSort
collatex
jingtrang
openmldb
BotCrypt
crate-anon
openlabs_production_disassembly
cubicweb-skillmat
django-shrink
Jiyatesting
MLTestServier
py-simple-ttk
urlscan-py
hack-assembler
git-todo
redbeaver
passninja
dbss-anisimov
Products.Quills
mopti
ippy
surveyhelper
timenazi
packagetest311110
Facebook_PyBot
ConfArgParse
himama
gdtools
py-nifcloud
ansys-mapdl-core
sphinxcontrib-cmtinc
cp-mn-bd-01-mimicat
pycompadre
sandatetime
octave-kernel
adafruit-circuitpython-httpserver
darkmixer
baizhansuperman
ASUSCloudInfra
teamrubber.recipe.portbase
rrrep
cattledb
cyPyon
python-parsely
spacy-universal-sentence-encoder
matop
kural
pyReachAPI
zuolar_nester
mrpeace
tap-responsys
PyGNS3
nsga2
pywkpdf
required
weatherpy2
pycopy-email.errors
py-telegram-notifier
pycosat
re-ver
swatchtime
let
subredis
mwdb
basic-instagram-library
falcon-signed-requests
collective.slideshare
pexpect-serial
epipy
messanger_project_server_parts
spiderdog
atilla
genepi
firehose
mc-man
PythonPluginFramework
invoicing-pdf-aa
eea.google
feature_eva
maestral
rapid
SQLAlchemy-Fixtures
pynspector
sasvenv
smartbot
chatbotentities
hn_sum
SFlock2
Products.CMFDynamicViewFTI
ASE-QE
sos-bash
thunderdome-flask
supermathaaabbb
cubicweb-preview
django-prospect
testpackageBP
picuplib
rhumbix-csv-uploader
ez-pg
pyf
prei
git-sync-notify
nsi.granulate
image-splitter
whaaaaat
plycounter
anyvc
cloudspot-license-api
distributionGauss
FragPELE
plover-phenrsteno
tinycert
libpython-do-pypi
streamlit-auth0-component
addlyrics
nrf24lu1-boot
icemac.ab.importxls
bfi
mesa-toolkit
twittertail
geox-statistics
aleph-lang
doublify-toolkit
torch-tensorrt
django-remote-submission
python-ntlm
asmjit
routing-engine
JGB-distributions
sed2
flask-whooshee
django-apiblueprint-tests
cpuspinner
django-better500s
autoarchive
vnpy-tinysoft
modelwhaleutils
Aman
Gwilio
gomobile.imageinfo
crcache
vestory
UFC
L-System-Visualizer
strictconf
libcredit.py
mixem
PrettyWrappers
pytest-timer
gptgladiator
keytar
udmlprob-distributions
deephit
bdot
bupt-session
pulumi-mysql
collective.linkedin
pmr.wfctrl
ctbBio27
uniproxy-mssngr-tools
tap-amazon-ads-dsp
Logs
checkmk-commander
diana-calc
django-freeplay
ezdashboard
ng.app.openid
django-prefixed-textinput
tensorflow-cpu-2.0-preview
simple-sqlite
dsn-mongo-wrapper
scikits.odes
github-create
pytest-reana
cliff-rackspace
mua-keras-models
pktt
Ajith-Kumar-Ravichander
django-static
lynx
fit
wepay-signer
dcd-sdk
PyDecimal
SQLAlchemy-Continuum-Trialspark
verktyg
ipyaggrid
sciencebeam-parser
pyalphabetica
retry-requests
snfatigue
kooPy
ginkgo-yan
elyzer
senaite.abx
geodjango-tigerline
soundsig
blatann
MegaScreen
ressl
scipion-app
collective.brasil.vocab
dsnd-probability-awsedp
cvbinarycanceraiopslibs
eulfedora
items-cooker-search
databricks-helper
versile-python3
sandstorm
shuffled-stats
random_color
Versio
goo
magneto
QuasarModel
typingdict
os-android-package-name-changer
micropython-parser
core-test
h2o-pysparkling-scoring-3.5
django-dirtyfield
tiddlywebplugins.ibuilder
django-steamauth
kswutils
ujson
Vicar2Png
bmd-py
disuniv
micropython-senko
txdlo
django1.11-zebra
pyalgotrade-groom
pydatum
fim-python
django-core-api
django-mysql-pool
mglib
lightpack
feed2tweet
slack-bot-wrapper
miquido-sts
spacenavigator
django-staticfiles-jquery
Presentations
td-ameritrade-api
SaltTesting
experiment-fsm
pip-packageexample
twod.wsgi
compares
djangologinapp
ajenti.plugin.core
libpythonpro-marcosviana
dzenlog-text
python-firebase
duffel
django-query-tools
Adafruit_AS726x
django-material-admin
panjing
collective.carouselbehaviour
validate_email
agottani-distributions
supermutes
zq-logger
lipidhandler
domain
sumoftwonumbers
Talaria
asuds
GitflowDennis
django-qshop
asqlite3
congo-red
hhic
django-sp
django-kong-admin
lovely.tal
django-extra_keywords
metriccounter
yamllib
bobsleigh
ps.diazo.realia
django-group-user-mngt
bee-django-crm
jitterentropy
WOPkg
Pong-helloworld-library
easykivy
opt
coco-tools
pydiggs
django_counters
djc.recipe
itamake
motional-ml-evaluate
ia256utilities
memcacheinspector
PySide6-FluentUI-QML
ansible-vault
escrotum
ergodic
agsi
mosviz
ptfutils
django-testproject-gito
whatever-rest-framework
rabbitsketch
Vocabulary
dbb_de_nester
abstention
ami-builder-api
migreat
django-duration-log
detectron2-cdo
sarf-zeromq-bridge
SIIM2_Server
ieml
entropy-framework
celare
gclid-timestamp-decoder
avalares
mara-app
thulac_test
vivarium-cobra
h2o-pysparkling-scoring-3.1
impress
enta
IOcode
scepia
sbo-sphinx
thatool
chemparse
FishFishJump
wireframe2html
xddos
nectarifere
patchit
apostle
accurate-timed-loop
foreground_app_info
gampc
pyabs
sparkpost-ip-notifier
cnd-utilitybelt
django-auth-fogbugz
neuralart
lc-pytools
python-labs-helpers
mutalyzer-hgvs-parser
lessrpc_msgpack
django-plugins-bihealth
taipy-rest
tazlib
aconfig
stratumgs
xx-swahili-spacy
micropython-array
correios
thread_processor
quast
vasprun-xml
django-concurrenflict
numba-kdtree
pytvmaze
arxiv-marxdown
document-transformers
inferno-pytorch
antigravity
tigopesa
benchlog
UrlMark
django-careers
scif
decompose-mrobison
remote-actions
query-exporter
pyfaceb
alt_cinder_sch
ffmpymedia
fightingcv
diceseq
wwo-hist
reacttable
beam-mysql-connector
fpkem
matrix-ops
django-authy
qis
autodocumentation
ogr
resaspy
echopype
tamarco-kafka
configloader
bobos3
structprop
pyramid_restful_toolkit
zgeo.recipe.openlayers
neuralnetwork
gmpy_cffi
netflix-spectator-py
buoyantPimpleFlux
pyphrasy
coordTransform
pickle-database
recordcapture
twisted-json-socket
Bconvertacceslib
tapioca-bitbucket
monkey
ansible-modules-dcos
coyote
sphinx-gettext-helper
flake8-deprecated
newscenter
secp256k1-zkp-mw
ddt-kion
manhattan-formatters
simpleservices
pp-transformers
RethinkDB_RDB
contributors-txt
sqlite3-plus
pygov
easyclasses
schema2rst
PyP6Xer
rattlesnake
audit-pkg
lcdevops
django-dynamic-models
vsav
letter-tools
pyrotash
TorchSUL
colorqrxy
pymatviz
stockx-py-sdk
torch-metrics
django-eztables
word-ps
fool
Halwa
upstride-argparse
eevee-flareon
frisk
judgmentModel
jingen
loopimer
butter
horusner
jsonnet_cpp11
rusty-axe-bbrener1
SBMLLint
exoskelegram
MLDateTimeParser
hse-ruz
django-uwsgi-admin
infi.bunch
mail_diff
jen
weldx-widgets
Oasys-Canvas-Core
rximp
MatPy
alphaqso
pyobjc-framework-PreferencePanes
pycopy-libc
elastic-transpor
stormtracks
also
Redis-Semaphore
allianceauth-imicusfat
Products.introspector
deepvision-toolkit
fineart-superset
linksgql
jungle
backrefs
humor-langs
multilabelencoder
camerata
viablimp
distributions-testupload-Udacity
pypinn
sandbox-lite-runner
scanjobs
gstreamer-rgw-sink
basevar
Topsis-Jashan
nmdriver
requests_extension
GraviPy
pytest-pride
py-googlesheets-grading
bestbuy
clitics-handler
cs2cpp
portiapy
progress_meter
moduloif
django-comuni-italiani
StockTrackr
stomper
pyclewn
pinnstf2
py-register-machine2
allianceauth-app-utils
yandex-torch
django-flash-rest
libsousou
helloWorld-nb-extension
kyqacommon
deepstack-python
surroopt
don
sqlalchemy-models
trader-client-kit
klok
gerenciamento-software
WordTreeGenerator
pipelineagents
AzurePyProxy
cloudshell-tg-breaking-point
adafruit-circuitpython-ht16k33
osmiter
fab
ceptacle
timeslice
cytoolz
umbrella
loadscreen
pycat3
collective.cicero
docassemble.hmctshelper
TyrionsCock
pgs-climbing
jabbergram
nifti-snapshot
fake-persian-name
mypackage-bod
ec2cluster
qtable
sentic
phash-blockhashio
judge
Tagineer
shoobx.wfmc
nos
meow-base
dashtable
score.webassets
ec2hashcat
mezzanine-recipes
uniborg
gcc_subnets
pyfuscate
ybEmail
dash-ant-design
mnist-pipeline
girder-hdfs-assetstore
oauth1-py3
pytorch-semseg
python-data-reducer
hammerspace
lunatic
pokeball
pysbrl
ruamel.zip2tar
seongjaeryu
CipherProgram
mymod1v1KLQ
python-aiml
pythonmarketo
rpki-as0-bogons
colorpennester
orcidfind
app-configuration
yandex-quadkey
zdup
tux_oracle
chances
Python-For-Excel
building
hellofrommoshe
cdsodatacli
automatron-sql
django-tekextensions
apply-argument-parser
panther-utils
Flask-Stripe
metro_distribution_engine
rsonlite
airwing_geoinfo
basicdweet
shcol
fastVAR
camprot
micropython-basicdweet
ConfigObj-GUI
ansible-lint-to-junit-xml
pynocaptcha
istart
gaus-bin-dists
plone.sequencebatch
pysas
geoai
postprocess-sdist-make-rust-ext-opt
ResponsiveTkinter
kinto-github-token
nl-dotter
ssrr
fundamental-distributions-gaussian-and-binomial
uhttplib
proteusisc
django-cqrs
meta1testnet
poppy
keragan
svglue
timeshade
sqlalchemy-hcm-postgresql
bodhi-messages
wdb-utils
liontoolkit
downloader-cluster
GDBIGtools
jk-asyncio-logging
pyzmo
seismicpy
peakrdl-systemrdl
sufi-dists
plone.formwidget.geolocation
cudnn-python-wrappers
toy-gradlogp
pycheesi
adi.trash
Augur
audiosync
staffeli
intra42
s3lncoll
usb-joc
glowbit
pyobjc-framework-WebKit
lv-probability
confire
opp
physiocurve
pylocale
NewLambData
django-multiple-include
graphflow
collective.js.formalize
linkedin-scraperb
ipymaterialui
glamkit-fallbackserve
libgeosuiteprv
manuell_nester
pyNmonAnalyzer
mkdocs-redirects
hytools
labs
python-instagram-2
tkCharacter
curses-playlist
veb
qutie
pyseoanalyzer
xs-authserver
vaultkeeper
dict-config-parser
querystream
bleach-allowlist
django-viewclass-mixins
zc.mappingobject
flex-format
silva.security.overview
djangocms-usersettings2
zappa-file-widget-mediabucket
tenable-ibm-cp4s
suds-jurko
warmap
pbnj
pycdr
datamap
tetris_cpp
publish-markdown
yargs
cogreqs
microbase
islpy
z3c.schemadiff
topsis-101803008
wanabrain
django-canonical-amp
modeltree
zfs_backup
staticwebsync
yandex_tools_wf_ws
users-pipeline
counting
eceibs
FYS2130-FreqAn
ssm-simulators
distribution-pkg
oauth-cli
ps-probability
django-native-datatables
freemysql
noseenv
pingping
mung
pattern-singleton
ros-cdk-hologram
cdmpyparser
twopi-flask-utils
DjangoVooster
allplay
jizhi
SSHLibrary
pq
saltedpwd
testproject1234
django-chosen-mega
jkinpylib
balkan-modules
ttslearn
website_status
yjpainting-sdk
data-stats-distributions
junitparsershubhindia
pysnip
fqn-decorators
python-workers-kv
space-bandits
newtonraphson
dcicpyvcf
booknlp
botchat
firstuploaddistributions
makeflaskreact
my-first-helloworld-script
cpthook
4ch
pling
graph-walker
pytroll-monitor
WB
minipot
odictliteral
humanleague
pyramid-redirect
ewb_case.portlet.get_involved
bin-gauss-distribution
findStr
kerasparalleltext
jrun
catnamescli
pytools123456789
djmicrosip-enviar-ventas
unshred
fxtest
toutpt.zopeskel
psdash
faas-grip
mlswarm
neotiles
orcanet
wrainfo
alpha_quant
Pat
pythongrid
clear-screen-CLS
wagtail-modelchooser-1.13.1
injectool
bob.db.avspoof
pingouin
neurosynchro
ntnu-process-mining
manimlib
LowCostSmartFarmHub
mfe_fwaas
GeneralRedditBot
Eaow
simple-dependencies-manager
spot-sdk
korean-lunar-calendar
urlextract-py2.7
urwide
xmisc
logic
sslcrypto
absp-socket
PyWorker
blueberry
resource-manager
bendercito
sphstat
ga4gh-server
izh2114
homeassistant-braviarc
tgapp-activitystream
vaex-server
simp_le
electiondata
pycopy-uos2
sessioner-globally.scoped
headmouse
Consul
django-theherk-menu-icons
utp
pytango-db
Data-Transformer
core-dashboard-app
Heliacal
AppleSearchAdsSDK
objtrack
simpledns
appomatic_photolog
parallel-queue
pyfingerd
python-mms
mmsplice
tumblr
ad-geo-backend
afew
django-sem
pywind
nester-blusili
PyParseBio
trelawney
pydatavolley
OptmizeExport
rand-useragent
mymod
project-migration
birdy
jsongrep
autorec
Solgema.EnvironmentViewlets
rip
django-seo-plus
JPMyNumber
sandy2pop
pipwhip
Wayne-381654729
ps
piping
django-mailer
llamirhosseinll
authok-python
sanic-macrobase
pigshare
httpie-hmac-auth
dns-sprockets
Sherbet
partial.py
eigency
asyncbot
tarantoolup
weka
midi-transformer
oftools
thomas-distributions
elasticopt
graphene-django-optimizer-patch
tide-query
microjoy
faste
toolbox-joacripp
ecmcli
lazyearth
lanscan
strec
django-surround
collective.behavior.relatedmedia
okcmd
braindynamics
hue
genenga
shells-cae
bitprophet
mf3cc
mezzanine-wymeditor
stompy
klue-microservice-async
techops.webforms
upfront.foldercontents
pybotlib
hypervector
lichvPy3mysql
golix
rok
TestFile
django-roles-access
mtrs-noodle
Products.Quaestrio
pymontasir
manish-package
merge_json
MultifunctionTools
nesterpack
dot_snek
My-Serializer-For-Json-And-XML-For-Lab3-3
de9im
django-auth-cli-certic
pyhash
smaz-py3
djangocommonhelper
matfactor
docker-secrets
dtmpy
apicast
aliyun-python-sdk-sts
nameless-deploy-tools
teeth-rest
dspca
Products.TemplateFields
Voltra.api
opencellid
dash-id-utils
noteworthy-notectl
ivi
sentiment_analyse
megrok.nozodb
django-always-authenticated
bitpy
nosql-versioning
adafruit-circuitpython-ble-apple-notification-center
edge
Salvia
conoha
rpi-dht22-exporter
stomp.py
cubicweb-file
openerp-web-view-editor
Demo-DAWGI-HBM
py-open-fonts
django-style-guide
pcommerce.shipment.pickup
geohashcx
AuthMe
celery-mongodb-nobinary-backend
botbuilder-sangam-twilio-whatsapp-adapter
BloxPy
daemon-runner
django-nap
jani
dbtpl
PyTablePrinter
flask-ml-client
djmicrosip_polizas
singleton3
django-inplaceedit-version1
```

Reply:
Some more just for reference

Reply:
added `pycosat` instead of helium

Reply:
hmm. that also needs a C compiler. Why not pick something _really_ lightweight? :)

Reply:
Hmm so that is using the system compiler in the example?

Reply:
What is a really light weight one then?